packages/client/hmi-client/src/components/dataset/tera-column-info.vue

```diff
@@ -0,0 +1,208 @@
+					:options="Object.values(ColumnType)"
+				/>
+			</span>
+			<span v-if="showConcept" class="concept">
+				<label>Concept</label>
+				<template v-if="featureConfig.isPreview">{{ query }}</template>
+				<AutoComplete
+					v-else
+					size="small"
+					placeholder="Search concepts"
+					v-model="query"
+					:suggestions="results"
+					optionLabel="name"
+					@complete="async () => (results = await searchCuriesEntities(query))"
+					@item-select="
+						$emit('update-column', { key: 'concept', value: { curie: $event.value.curie, name: $event.value.name } })
+					"
+					@keyup.enter="applyValidConcept"
+					@blur="applyValidConcept"
+				/>
+			</span>
+			<span class="description">
+				<template v-if="featureConfig.isPreview">{{ column.description }}</template>
+				<tera-input-text
+					v-else
+					placeholder="Add a description"
+					:model-value="column.description ?? ''"
+					@update:model-value="$emit('update-column', { key: 'description', value: $event })"
+				/>
+			</span>
+		</section>
+		<tera-boxplot class="flex-1" v-if="column.stats" :stats="column.stats" />
+	</div>
+</template>
+
+<script setup lang="ts">
+/* Copied the structure of tera-model-parts.vue */
+import { ref, computed, watch } from 'vue';
+import TeraInputText from '@/components/widgets/tera-input-text.vue';
+import TeraBoxplot from '@/components/widgets/tera-boxplot.vue';
+import AutoComplete from 'primevue/autocomplete';
+import Dropdown from 'primevue/dropdown';
+import { type DKG, ColumnType, type Grounding } from '@/types/Types';
+import { searchCuriesEntities } from '@/services/concept';
+import type { FeatureConfig } from '@/types/common';
+
+type ColumnInfo = {
+	symbol: string;
+	dataType?: ColumnType;
+	description?: string;
+	grounding?: Grounding;
+	// Metadata
+	unit?: string;
+	name?: string;
+	stats?: any;
+};
+
+const props = defineProps<{
+	column: ColumnInfo;
+	featureConfig: FeatureConfig;
+}>();
+
+const emit = defineEmits(['update-column']);
+
+const query = ref('');
+const results = ref<DKG[]>([]);
+
+// If we are in preview mode and there is no content, show nothing
+const showConcept = computed(() => !(props.featureConfig.isPreview && !query.value));
+
+// Used if an option isn't selected from the Autocomplete suggestions but is typed in regularly
+function applyValidConcept() {
+	// Allows to empty the concept
+	if (query.value === '') {
+		emit('update-column', { key: 'concept', value: { curie: '', name: '' } });
+	}
+	// If what was typed was one of the results then choose that result
+	else {
+		const concept = results.value.find((result) => result.name === query.value);
+		if (concept) {
+			emit('update-column', { key: 'concept', value: { curie: concept.curie, name: concept.name } });
+		}
+	}
+}
+
+watch(
+	() => props.column.grounding?.identifiers,
+	async (identifiers) => {
+		// console.log(identifiers); // FIXME: Multiple identifiers are held in here after enrichment! Designs have to be updated to handle more.
```

Comment:
Not a big deal:
If there is a ticket assigned for these designs/fix it would be helpful to have it here so this doesnt get lost?